docroot/modules/custom/va_gov_backend/src/Service/UserPermsService.php

```diff
@@ -130,6 +136,12 @@ public function userOptionsStorage(array &$form, array $targets, $user_id) {
   public function userAccess($entity_id, $entity_type, $user_id, $op = NULL) {
     $account = $this->getUser($user_id);
 
+    // When using entity reference view as a source for field options,
+    // first option has key '_none' and should be ignored in access checks.
+    if ($entity_id == '_none') {
```

Comment:
Minor: it's preferable to use strict comparison checking to loose, e.g:
```suggestion
    if ($entity_id === '_none') {
```